teuthology/task/install/__init__.py

```diff
@@ -441,6 +441,18 @@ def task(ctx, config):
              - url: "http://location.repo"
                name: "ceph_repo"
 
+    Add repo before trying to install any package, the shaman
+    related tasks will be ignored.
```

Comment:
Add repo before trying to install any package (all Shaman-related tasks will be ignored):